frame/fee-market/src/lib.rs

```diff
@@ -151,22 +149,32 @@ pub mod pallet {
 	// Priority relayers storage
 	#[pallet::storage]
 	#[pallet::getter(fn assigned_relayers)]
-	pub type AssignedRelayersStorage<T: Config> =
+	pub type AssignedRelayers<T: Config> =
 		StorageValue<_, Vec<Relayer<T::AccountId, RingBalance<T>>>, OptionQuery>;
 
 	// Order storage
 	#[pallet::storage]
+	#[pallet::getter(fn order)]
 	pub type Orders<T: Config> = StorageMap<
 		_,
 		Blake2_128Concat,
 		(LaneId, MessageNonce),
-		Order<T::AccountId, T::BlockNumber, Fee<T>>,
+		Order<T::AccountId, T::BlockNumber, RingBalance<T>>,
 		OptionQuery,
 	>;
 	#[pallet::storage]
 	pub type ConfirmedMessagesThisBlock<T: Config> =
 		StorageValue<_, Vec<(LaneId, MessageNonce)>, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn reward_mode)]
+	pub type RewardModeStorage<T: Config> =
```

Comment:
```suggestion
	pub type RewardMode<T: Config> =
```

Reply:
The enum is named `RewardMode`, So the storage can't the same as the enum.

Reply:
Why not use `MaxSlashableCollateral` as I suggested, that would be more simple and clean.

Reply:
`MaxSlashableCollateral` is easy to confuse with an order's locked collateral(the max slash value for an order). Particularly we have introduced the S(t) function to calculate the conrete slash value.

Reply:
It is naming issue, let's choose a better name then.

How about `CollateralSlashProtect`

Reply:
I prefer `CollateralSlashMode`, In normal cases, nothing related to protection, right?

Reply:
Done